gradle/lib/dependabot/gradle/version.rb

```diff
@@ -153,6 +153,10 @@ def pad_for_comparison(prefixed_tokens, other_prefixed_tokens)
       end
 
       def compare_prefixed_token(prefix:, token:, other_prefix:, other_token:)
+        return 1 if token == "+" && other_token != "+"
+        return -1 if other_token == "+" && token != "+"
+        return 0 if token == "+" && other_token == "+"
+
```

Comment:
This is probably a bit gnarly, and not recommended, but I think `other_token <=> token` will produce the desired results, it relies on `"1" > "+"` being true

Reply:
Yeah that might be misleading, although a part of me likes it. I'll stick with the long-winded version for now.